### PR TITLE
Fix hipEnvVar test

### PR DESCRIPTION
**Pull request by sunway513**
This PR fixes indentation, binary path and adding wait on popen calls for the hipEnvVarDriver test.

### CODE REVIEW

**Comment by kknox** [comment]
You need to [adjust the jenkinsfile](https://github.com/ROCm-Developer-Tools/HIP/blob/master/Jenkinsfile#L122) to include the unit test again

**Comment by sunway513** [comment]
Thanks Kent, updated in the PR ☺

Regards,
Peng

File: Jenkinsfile [hunk 1]
```diff
@@ -119,7 +119,6 @@ def docker_build_inside_image( def build_image, String inside_args, String platf
             cd ${build_dir_rel}
             make install -j\$(nproc)
             make build_tests -j\$(nproc)
-            ctest -E hipEnvVarDriver
```
Comment (kknox) [inline Jenkinsfile]: I think you just deleted all testing :)  You probably need to replace with 'make test'

Comment (sunway513) [inline Jenkinsfile]: Thanks Kent :-) Updated with "make test".

**Comment by kknox** [comment]
hipEnvVar failed on nvidia machine.  You can look at the console output by clicking the red X above.

[nvcc] + make test
[nvcc] Running tests...
[nvcc] Test project /var/jenkins/workspace/-Developer-Tools_HIP_PR-110-DAATY7BOJMZLIC2LA357DNAY74R3BVG2A5MZS37A7WME75TYOZLA/build/release
[nvcc]       Start  1: directed_tests/hipEnvVarDriver.tst
[nvcc]  1/71 Test  #1: directed_tests/hipEnvVarDriver.tst ........................................***Failed  Required regular expression not found.Regex=[PASSED
[nvcc] ]  0.13 sec

**Comment by bensander** [comment]
Peng is this still ready to be accepted?

**Comment by sunway513** [comment]
Hi Ben, still working on NV path fix, will push it soon.

**Comment by sunway513** [comment]
@bensander , can you review?

**Pull request merged**